### PR TITLE
feat: integrated left content

### PR DESCRIPTION
**Pull request by hardisp**
This PR for integrated the left side with real data.

### CODE REVIEW

File: server/main.js [hunk 1]
```diff
@@ -58,8 +58,8 @@ Empirica.gameInit((game, treatment, players) => {
   const feedback = game.treatment.giveFeedback || false;
   // const stageDuration = game.treatment.stageLength || 120;
   // const socialStageDuration = game.treatment.socialStageLength || 120;
-  const stageDuration = 50000;
-  const socialStageDuration = 2000;
+  const stageDuration = 3500;
+  const socialStageDuration = 3500;
```
Comment (npaton) [inline server/main.js]: We should restore the original stage duration.

File: client/game/Timer.jsx [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+  startTimer = () => {
+    const { remainingSeconds } = this.props;
+    const timer = remainingSeconds;
+    let minutes;
+    let seconds;
+    minutes = parseInt(timer / 60, 10);
+    seconds = parseInt(timer % 60, 10);
+
+    minutes = minutes < 10 ? "0" + minutes : minutes;
+    seconds = seconds < 10 ? "0" + seconds : seconds;
```
Comment (npaton) [inline client/game/Timer.jsx]: You can use padStart for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

File: client/game/Timer.jsx [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
 import Timer from "./Timer.jsx";
 
 class timer extends React.Component {
+  startTimer = () => {
+    const { remainingSeconds } = this.props;
+    const timer = remainingSeconds;
+    let minutes;
+    let seconds;
+    minutes = parseInt(timer / 60, 10);
+    seconds = parseInt(timer % 60, 10);
```
Comment (npaton) [inline client/game/Timer.jsx]: You can out initialisation on same line as declaration here.

```suggestion
    let minutes = parseInt(timer / 60, 10);
    let seconds = parseInt(timer % 60, 10);
```

File: client/game/TaskStimulus.jsx [hunk 4]
```diff
@@ -1,64 +1,111 @@
...
-          ""
-        )}
+      <div className="couples">
+        <div className="person">
+          <PersonCard pairData={pairData} isPartner />
+          <div className="ratings">
+            <h3>Ratings</h3>
+            <RenderFeatureRow pairData={pairData} isPartner />
+          </div>
+        </div>
```
Comment (npaton) [inline client/game/TaskStimulus.jsx]: Looks like all this code is repeated and can go into PersonCard.

**Review by npaton — CHANGES_REQUESTED** [review]
LGTM, just a couple minor changes 👍

File: client/game/Timer.jsx [hunk 5]
```diff
@@ -8,14 +8,16 @@ class timer extends React.Component {
...
+    let seconds = parseInt(timer % 60, 10);
 
     minutes = minutes < 10 ? "0" + minutes : minutes;
     seconds = seconds < 10 ? "0" + seconds : seconds;
-    return minutes + ":" + seconds;
+    return (
+      minutes.toString().padStart(2, "0") +
+      ":" +
+      seconds.toString().padStart(2, "0")
+    );
```
Comment (npaton) [inline client/game/Timer.jsx]: You're doing the same thing twice here.

**Review by npaton — CHANGES_REQUESTED** [review]

**Review by hubertusputu1 — APPROVED** [review]

File: client/game/Timer.jsx [hunk 6]
```diff
@@ -11,8 +11,8 @@ class timer extends React.Component {
     let minutes = parseInt(timer / 60, 10);
     let seconds = parseInt(timer % 60, 10);
 
-    minutes = minutes < 10 ? "0" + minutes : minutes;
-    seconds = seconds < 10 ? "0" + seconds : seconds;
+    minutes = minutes < 10 ? minutes : minutes;
+    seconds = seconds < 10 ? seconds : seconds;
```
Comment (npaton) [inline client/game/Timer.jsx]: These 2 lines aren't doing anything now...

**Review by npaton — CHANGES_REQUESTED** [review]

**Review by npaton — APPROVED** [review]

**Pull request merged**